pkg/asset/installconfig/aws/availabilityzones.go

```diff
@@ -90,14 +90,27 @@ func zonesByType(ctx context.Context, session *session.Session, region string, z
 	return zones, nil
 }
 
-// availabilityZones retrieves a list of zones type 'availability-zone' for the region.
+// availabilityZones retrieves a list of zones type 'availability-zone' in the region.
 func availabilityZones(ctx context.Context, session *session.Session, region string) ([]string, error) {
-	return zonesByType(ctx, session, region, typesaws.AvailabilityZoneType)
+	return filterZonesByType(ctx, session, region, typesaws.AvailabilityZoneType)
 }
 
-// localZones retrieves a list of zones type 'local-zone' for the region.
-func localZones(ctx context.Context, session *session.Session, region string) ([]string, error) {
-	return zonesByType(ctx, session, region, typesaws.LocalZoneType)
+// edgeZones retrieves a list of zones type 'local-zone' and 'wavelength-zone' in the region.
+func edgeZones(ctx context.Context, session *session.Session, region string) ([]string, error) {
+	localZones, err := filterZonesByType(ctx, session, region, typesaws.LocalZoneType)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to retrieve Local Zone names: %s", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("unable to retrieve Local Zone names: %w", err)
```
The errors package is deprecated and the project has been archived: https://github.com/pkg/errors